core/src/bitmap/operations.rs

```diff
@@ -1201,6 +1201,171 @@ pub fn apply_filter<'gc>(
     }
 }
 
+#[allow(clippy::too_many_arguments)]
+fn copy_on_cpu<'gc>(
+    context: &mut UpdateContext<'_, 'gc>,
```

Comment:
This only ever uses a `MutationContext`- would it be (slightly) more performant to take a `MutationContext` over an `UpdateContext`?

Reply:
Changed!